main.js

```diff
@@ -32,10 +32,10 @@ app.whenReady().then(() => {
   })
 })
 
-// Quit when all windows are closed.
+// Quit when all windows are closed, except on macOS. There, it's common
+// for applications and their menu bar to stay active until the user quits
+// explicitly with Cmd + Q.
```

Comment:
i'm not sure what's changed here? this copy was just moved but doesn't look to have been changed at all

Reply:
It's basically clarification via consolidating the comments. See https://github.com/electron/electron/pull/24003 for context.